src/features/main/components/WhyWeCareSection/WhyWeCareSection.tsx

```diff
@@ -0,0 +1,33 @@
+import Box from '@mui/material/Box';
+import DetailsCard from '../DetailsCard/DetailsCard';
+import { cardsData, title } from './constants';
+import Title from '../../../../components/Title/Title';
+import classes from './whyWeCareSection.module.scss';
+
+export default function WhyWeCareSection() {
+  return (
+    <Box component="section" className={classes.section} id="why-we-care">
+      <Box
+        component="div"
+        sx={{
+          maxWidth: { lg: '1108px', md: '850px', xs: 'fit-content' },
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 2,
+          margin: 'auto',
+          py: 12,
+        }}
+      >
+        <Title title={title} />
+        <Box
+          component="div"
+          sx={{ display: 'flex', flexDirection: { md: 'row', xs: 'column' }, gap: 2 }}
```

Comment:
For this wrapper need use rule justify-contetnt: space-between for containing all width of container.
Now you have free space on right side.
```suggestion
          sx={{
            display: 'flex',
            justifyContent: 'space-between',
            flexDirection: { md: 'row', xs: 'column' },
            gap: 2,
          }}
```